bin/bitbake-setup

```diff
@@ -170,11 +171,11 @@ def setup_bitbake_build(bitbake_config, layerdir, builddir):
     if template:
         bb.process.run("{} setup -c {} -b {} --no-shell".format(oesetupbuild, template, bitbake_builddir))
     else:
```

Comment:
Thanks for the patches. Before I can accept them, this bit needs to be fixed, e.g. the case when the build/conf directory is not populated by bitbake-setup, but rather by oe-setup-build. Otherwise dash will work only in one case out of the two.

So can you run oe-setup-build directly, see what it does, whether it can be used with dash, and send any needed patches to oe-core?

Reply:
patch sent to oe-core: https://lists.openembedded.org/g/openembedded-core/message/224083